src/common/MovieTile/index.js

```diff
@@ -35,6 +36,10 @@ const MovieTile = ({ id, poster, title, character, year, genre, rating, votes })
 				<CharacterAndYearElements>
 					<Character>{character ? character : ""}</Character>
 					<Year>{year ? (new Date(year).getFullYear()) : ""}</Year>
+					<YearInParentheses>
+						{yearInParentheses ?
+							`(${(new Date(yearInParentheses).getFullYear())})` : ""}
+					</YearInParentheses>
```

Comment:
Dzięki, jakoś pominąłem to, żeby rok był  w nawiasie 👍

Reply:
Spoko, ja przy reviewie też nie zauważyłem :)